tests/expectations/test_expectation_atomic_renderers.py

```diff
@@ -160,6 +162,10 @@ def test_atomic_prescriptive_summary_expect_column_distinct_values_to_equal_set(
     snapshot.assert_match(res)
 
 
+@pytest.mark.skipif(
+    is_version_greater_or_equal(pd.__version__, "2.0.0"),
+    reason="Altair is currently not compatible with pandas 2.0.0",
```

Comment:
Can we make these three tests xfail strict instead of skip if the pandas version is >2.0.0? That way when altair is compatible these tests will get updated or re-enabled.

Reply:
Good call. Updating now

Reply:
But won't this xpass in parts of our CI? Which means it fails if it's a `strict` xfail

Reply:
Oh is that the case @kilo59? Is there a way to do a "conditiona" xfail?

Reply:
Found it:  I think we can be more specific with the xfail as @anthonyburdi is describing. Will update / run the pipeline with the described changes.

Reply:
@Shinnnyshinshin If we are saying that this "test could fail and that's ok" that's just normal `xfail` `strict=False`.

As @anthonyburdi said `xfail(strict=True)` can be useful if we want to force some code change once the test no-longer fails.

But I would assume this test will `XPASS` in our CI when `pandas<2.0.0` right? And if that's the case we can't use `xfail(strict=True)` because it will fail the CI step.

If we are saying "this test only makes sense under this condition"  `skipif` is the right choice.